fdbclient/ClientKnobs.cpp

```diff
@@ -303,7 +303,7 @@ void ClientKnobs::initialize(Randomize randomize) {
 
 	init( ENABLE_ENCRYPTION_CPU_TIME_LOGGING,       false );
 	init( SIMULATION_EKP_TENANT_IDS_TO_DROP,         "-1" );
-	init( ENABLE_CONFIGURABLE_ENCRYPTION,           false );
```

Comment:
want to set it to `true` ?

Reply:
Yi did it in his pr, until we buggify this knob might was well keep it to true to test configurable encryption in nightlies